frontend/src/helpers/routes/is-route-have-header.ts

```diff
@@ -0,0 +1,5 @@
+import { RoutesWithoutHeader } from 'common/enums/app/app-route.enum';
+
+export const isRouteHaveHeader = (currentRoute: string): boolean => {
+  return (Object.values(RoutesWithoutHeader) as string[]).includes(currentRoute);
+};
```

Comment:
`RoutesWithoutHeader` is a enum, so it's a static data. You do not need to call Object.values each time when you call this method. Move it above function.